moveit_experimental/moveit_jog_arm/src/cpp_interface_example/cpp_interface_example.cpp

```diff
@@ -67,14 +70,14 @@ int main(int argc, char** argv)
   planning_scene_monitor->startStateMonitor();
 
   // Run the jogging C++ interface in a new thread to ensure a constant outgoing message rate.
-  moveit_jog_arm::JogCppInterface jog_interface(planning_scene_monitor);
-  std::thread jogging_thread([&]() { jog_interface.startMainLoop(); });
+  moveit_jog_arm::JogArm jog_arm(nh, planning_scene_monitor);
+  jog_arm.start();
 
   // Make a Cartesian velocity message
-  geometry_msgs::TwistStamped velocity_msg;
-  velocity_msg.header.frame_id = "base_link";
-  velocity_msg.twist.linear.y = 0.01;
-  velocity_msg.twist.linear.z = -0.01;
+  auto velocity_msg = boost::make_shared<geometry_msgs::TwistStamped>();
```

Comment:
Can you add a comment here for people like me (and probably 90% of C++ users) to clarify why you're using a shared pointer?

Reply:
Thank you for pointing this out.  I'd updated this in my profile to work correctly but hadn't updated here.  There is now a comment to go along with this too.